lib/x11utils.pm

```diff
@@ -237,4 +247,67 @@ sub turn_off_gnome_suspend {
     script_run 'gsettings set org.gnome.settings-daemon.plugins.power sleep-inactive-ac-type \'nothing\'';
 }
 
+=head2 turn_off_xfce_screensaver
+
+ turn_off_xfce_screensaver();
+
+Disables screensaver in xfce. To be called from a command prompt, for example an xterm window.
+
+=cut
+sub turn_off_xfce_screensaver {
+    script_run 'xfconf-query -c xfce4-screensaver -p /saver/enabled -n -s false';
+    script_run 'xfconf-query -c xfce4-screensaver -p /lock/enabled -n -s false';
+}
+
+=head2 turn_of_x11_dpms
+
+ turn_off_x11_dpms();
+
+This will disable the X11 Display Power Management Signaling
+eg. blanking the screen and sending the monitor to sleep after some time.
+
+This will disable the setting at runtime and also add a config file for the
+XServer to make this change persist reboots.
+
+This function must be called from an xterm - B<not> from a bare tty.
+This is because the tty will not have the permissions to run C<xset>.
+
+=cut
+sub turn_off_x11_dpms {
```

Comment:
Something similar could be applied to ttys? `consoleblank=0` via grub?

Reply:
I don't think this is needed as we already run `$self->script_run('setterm -blank 0')` from `lib/susedistribution.pm` in the `activate_console` method.